src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,65 @@
+package duke;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.time.format.DateTimeParseException;
+
+import duke.command.Command;
+import duke.exception.DukeException;
+
+/**
+ * The Duke Program is a chatbot that takes in
+ * input from the user and responds accordingly
+ */
+public class Duke {
+    private Storage storage;
+    private TaskList tasks;
+    private UI ui;
+    private String uiOutput;
+    private boolean justCreatedFile = false;
```

Comment:
As much as possible, use a prefix such as is, has, was, etc. for boolean variable/method names so that linters can automatically verify that this style rule is being followed.

```suggestion
    private boolean hasCreatedFile = false;
```